docs/scalars_and_keras.ipynb

```diff
@@ -440,28 +412,202 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 10,
+      "execution_count": null,
       "metadata": {
         "id": "97T4vT3QkQJH"
       },
-      "outputs": [
-        {
-          "name": "stdout",
-          "output_type": "stream",
-          "text": [
-            "[[32.234013 ]\n",
-            " [14.5973015]\n",
-            " [ 3.0074618]]\n"
-          ]
-        }
-      ],
+      "outputs": [],
       "source": [
         "print(model.predict([60, 25, 2]))\n",
         "# True values to compare predictions against: \n",
         "# [[32.0]\n",
         "#  [14.5]\n",
         "#  [ 3.0]]"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oxAVfW8lhZ0e"
+      },
+      "source": [
+        "## Batch-level logging\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "p-ltiSuypLVE"
+      },
+      "source": [
+        "First let's load the MNIST dataset, normalize the data and write a function that creates a simple Keras model for classifying the images into 10 classes."
```

Comment:
Yes, this can be done in the future. I kept it with MNIST to allow for the use of more diverse custom metrics used in multiclass classification, such as sparse categorical accuracy, but this can be simplified for consolidation in a later PR.